decks/Elections-Proposal.mdx

```diff
@@ -0,0 +1,65 @@
+import { CodeSurfer as Surfer } from "code-surfer";
+import { CodeSurferColumns, Step } from "code-surfer";
+import { Appear, Background } from "gatsby-theme-mdx-deck";
+import * as L from "../src/layout";
+import customTheme from "../src/theme";
+import GreetingLoader from "./src/greeting-loader";
+
+export const theme = customTheme;
+
+# Workshop Proposal
+
+---
+
+# Specific Topic
+- We are going to use the OpenFEC API Election endpoint to visualize data for total disbursements and total receipts for Presidential candidates for the last 30 elections.
+- Graph will compare this financial information over time and for both the Democratic and Republican parties. 
+
+---
+
+# Interactive Component
+- We will have students visualize data from the same endpoint but for cash_on_hand_end_period 
+---
+
+# Code Deliverables
+## Week 2
+- Code out total disbursements and total receipts (Work on together)
+
+## Week 3
+- Finish up any unfinished work from week 2 + complete interactive activity (Work on together)
+
+## Week 4
+- Finish up any unfinished code from interactive activity
+- Work on presentation slides (will decide how to split up)
+---  
+
+# Slide Outline
+What is a general roadmap for your slides?
+- Intro to tech (Chartjs, d3js)
+- Visualize information from API and Chartjs (also explain how to get the JSON from the website)
+- Explain and present our code
+- Present interactive activity prompt
+- Conclusion and why its important
+---
+
+# Timing
+Each workshop should be around an hour to 90 minutes. How long will your workshop be?
+- An hour
+---
+
+# Motivation behind project
+What motivated your duo to pursue this idea?
+- Explored the API and searched through to find interesting information
+---
```

Comment:
Hmm, encourage interest in a kid to care about fed election. why is visualizing financial data important? What is cool/interesting about it? Maybe compare how previous presidents have spent in correlation to their approval numbers, budget management...etc